package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "guitar-buddy-server",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www"
```

Comment:
`nodemon` is not in your dependency list so this `start` script only works for people who have `nodemon` installed globally. So it would be good to add it to `dev-dependencies`.